meta-refkit/recipes-image/images/refkit-installer-image.bb

```diff
@@ -23,10 +23,14 @@ REFKIT_INSTALLER_UEFI_COMBO () {
         rootfs=$4
         output_mounted=
         output_mountpoint=
+        output_luks=
+        LUKS_NAME=rootfs
+        LUKS_PASSWORD=refkit
```

Comment:
There should probably be a big warning message so that no-one will use this password accidentally.